Source/Core/Core/PowerPC/Interpreter/Interpreter.cpp

```diff
@@ -285,25 +285,25 @@ void Interpreter::Run()
           if (PowerPC::breakpoints.IsAddressBreakPoint(PC))
           {
 #ifdef SHOW_HISTORY
-            NOTICE_LOG(POWERPC, "----------------------------");
-            NOTICE_LOG(POWERPC, "Blocks:");
-            for (int j = 0; j < PCBlockVec.size(); j++)
-              NOTICE_LOG(POWERPC, "PC: 0x%08x", PCBlockVec.at(j));
-            NOTICE_LOG(POWERPC, "----------------------------");
-            NOTICE_LOG(POWERPC, "Steps:");
-            for (int j = 0; j < PCVec.size(); j++)
+            NOTICE_LOG_FMT(POWERPC, "----------------------------");
+            NOTICE_LOG_FMT(POWERPC, "Blocks:");
+            for (const int entry : PCBlockVec)
+              NOTICE_LOG_FMT(POWERPC, "PC: {:#010x}", entry);
+            NOTICE_LOG_FMT(POWERPC, "----------------------------");
+            NOTICE_LOG_FMT(POWERPC, "Steps:");
+            for (size_t j = 0; j < PCVec.size(); j++)
             {
               // Write space
               if (j > 0)
               {
-                if (PCVec.at(j) != PCVec.at(j - 1) + 4)
-                  NOTICE_LOG(POWERPC, "");
+                if (PCVec[j] != PCVec[(j - 1) + 4]
+                  NOTICE_LOG_FMT(POWERPC, "");
               }
 
-              NOTICE_LOG(POWERPC, "PC: 0x%08x", PCVec.at(j));
+              NOTICE_LOG_FMT(POWERPC, "PC: {:#010x]", PCVec[j]);
```

Comment:
Shouldn't the compile time checking hit this one?
```suggestion
              NOTICE_LOG_FMT(POWERPC, "PC: {:#010x}", PCVec[j]);
```

Reply:
`SHOW_HISTORY` isn't defined on the buildbot, so this code isn't checked.

Arguably, this code should be removed entirely since I don't think it's been used for ages, but that'd be something for a different PR.